### ISSUE TITLE
Dashlet Header Edit, Refresh, Delete links are not accessible by screen readers

### ISSUE THREAD

**Issue by kf4hav** [salesagility/SuiteCRM#2293]
On the screen after logging in, there are three links after the following headings.
My Calls
My Meetings
My Top Open Opportunities
My Accounts
My Leads
My Activity Stream
These three links are just above the tables showing the subsequent sections. With screen readers which rely on text to make people who are blind aware of links, these links read void(0). My on android tablet running android 5.1 with the latest release of chrome these links do not read at all. Unless I click on these links there is no way of knowing what they do. Since I am blind I have no idea if there are accompanying images to go with these links, but after looking at the source code it does not look like this is the case.
I have already found a fix for this issue, and I am willing to submit the fix so as to help make suite crm better for everyone. I just need to know if I need to submit to a specific branch since I directly cloned from
https://github.com/salesagility/SuiteCRM.git

**Comment by ijdavie** [comment]
@kf4hav, 
When committing back most of the information that you will need can be found in the following location 

https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM

The main thing is to make sure your pull request is on the hotfix branch. 

This is partly related to #1633. 
Thanks.
